packages/ui/src/__stories__/components/Colors.tsx

```diff
@@ -125,13 +141,165 @@ const Colors = () => {
                       </StyledCard>
                     </Stack>
                   ))}
-              </div>
-            </div>
+              </Stack>
+            </Row>
             <StyledSeparator />
-          </>
+          </Stack>
         )
       })}
-    </>
+
+      <Stack gap={2}>
+        <Text variant="heading" as="h2">
+          Other
+        </Text>
+        <Text variant="headingSmall" as="h3">
+          Data
+        </Text>
+        <Row templateColumns="repeat(3, 1fr)" gap={2}>
+          {Object.keys(dataColors).map(data => (
```

Comment:
You may be able to do  `((Object.keys(dataColors)) as  (keyof typeof dataColors)[]).map` to avoid multiple `as` later